.github/workflows/CI.yml

```diff
@@ -71,13 +71,14 @@ jobs:
       - uses: actions/checkout@v2
       - uses: julia-actions/setup-julia@v1
         with:
-          version: '1'
+          version: '1.6'
```

Comment:
I think this should probably be `1`, as opposed to `1.6`.

`1` means "the latest version in the 1.x series of releases", so at the moment it will give `1.6`. When `1.7` is released, it'll do that.

Reply:
Actually, this should probably just be `1` and `1.6`, so that we don't accidentally drop support for `1.6` before we want to.

Reply:
So you're thinking:
```
version: 
   - '1'
   - '1.6'
``` 
?

Reply:
Exactly.

Reply:
Done

Reply:
This seems to have broken the `CI`. Adding the second version must be more than that one liner.

Reply:
Resolving since discussed in a thread below